From c781008678fc330fd5ae8caab1c5aa986bafe298 Mon Sep 17 00:00:00 2001
From: Elimar Riesebieter <riesebie@lxtec.de>
Date: Fri, 14 Dec 2018 13:04:46 +0100
Subject: [PATCH] Fix an unescaped brace in amavisd

---
 amavisd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/amavisd b/amavisd
index 7f93194..70a8a28 100755
--- a/amavisd
+++ b/amavisd
@@ -28848,7 +28848,7 @@ sub run_av(@) {
         my $f = $bare_fnames->[$k];  my $multi = 0;
         if ($one_at_a_time) {  # glob templates may be substrings anywhere
           local($1);  @query_expanded = @query_template;  # start afresh
-          s{ ( {} (?: / \* )? | \* ) }
+          s{ ( \{\} (?: / \* )? | \* ) }
            { $1 eq '{}'   ? "$tempdir/parts"
            : $1 eq '{}/*' ? ($multi=1,"$tempdir/parts/$f")
            : $1 eq '*'    ? ($multi=1,$f)  : $1
-- 
2.20.0

