--- /etc/amavisd.conf
+++ /tmp/amavisd.conf
@@ -165,10 +165,12 @@
 
 # OTHER MORE COMMON SETTINGS (defaults may suffice):
 
-# $myhostname = 'host.example.com';  # must be a fully-qualified domain name!
+$myhostname = `hostname -d`;
+$inet_socket_bind = '0';
+@inet_acl = qw( 0/0 );
 
-# $notify_method  = 'smtp:[127.0.0.1]:10025';
-# $forward_method = 'smtp:[127.0.0.1]:10025';  # set to undef with milter!
+$notify_method  = "smtp:$ENV{POSTFIX_IP}:10025";
+$forward_method  = "smtp:$ENV{POSTFIX_IP}:10025";
 
 # $final_virus_destiny      = D_DISCARD;
 # $final_banned_destiny     = D_DISCARD;
@@ -812,11 +814,11 @@
 # You may also have to adjust the path to the clamd communication
 # socket to match your system. The path is usually defined in the file
 # /etc/clamd.conf, or may be controlled by your service manager / init.
-#
-# ['ClamAV-clamd',
-#   \&ask_daemon, ["CONTSCAN {}\n", "/var/run/clamav/clamd.sock"],
-#   qr/\bOK$/m, qr/\bFOUND$/m,
-#   qr/^.*?: (?!Infected Archive)(.*) FOUND$/m ],
+
+  ['ClamAV-clamd',
+    \&ask_daemon, ["CONTSCAN {}\n", "clamd:3310"],
+    qr/\bOK$/m, qr/\bFOUND$/m,
+    qr/^.*?: (?!Infected Archive)(.*) FOUND$/m ],
 
 # ['ClamAV-clamd-stream',
 #   \&ask_daemon, ["*", 'clamd:/var/run/clamav/clamd.sock'],
